src/refactor_nrepl/artifacts.clj

```diff
@@ -51,35 +51,42 @@
          ;; Ignore artifact if not readable. See #255
          nil)))
 
+(def ^:private clojars-artifacts-cache (atom nil))
+
 (defn get-clojars-artifacts!
   "Returns a vector of [[some/lib \"0.1\"]...]."
   []
   (try
-    (->> "https://clojars.org/repo/all-jars.clj.gz"
-         io/input-stream
-         GZIPInputStream.
-         io/reader
-         line-seq
-         (keep edn-read-or-nil))
+    (or @clojars-artifacts-cache
```

Comment:
I believe this breaks the `force` functionality in `artifact-list` which forces an update of a potentially stale cache.

Reply:
Mmm good once, I briefly considered the daily caching but it didn't seem too relevant.

It is for long-lived sessions which are very much emacsy :)